### PR TITLE
chore: adding new kpi report workflow

### PR DESCRIPTION
**Pull request by nagash77**
This KPI report workflow begins by generating reports on open vs closed issues and triage throughput metrics

### Steps to test

I don't think there is a way to test this easily.  I have been doing testing in my private repos to verify it works.

### PR Tasks

- [ n/a ] Have tests been added/updated?
- [ n/a ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ n/a ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

**Comment by nagash77** [comment]
relies on this PR in the release-automations repo - https://github.com/cypress-io/release-automations/pull/85

File: .github/workflows/report_weekly_app_kpis.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        required: false
+        default: '["cypress-documentation", "circleci-orb", "github-action", "cypress-docker-images", "cypress-chrome-recorder-extension", "cypress-chrome-recorder", "cypress-recorder-extension", "cypress-example-kitchensink", "cypress-origin-providers", "release-automations", "cypress-test-tiny", "eslint-plugin-cypress", "netlify-plugin-cypress", "cypress-support-internal", "cypress-realworld-app", "code-coverage" ]'
+      default-mono-repo:
+        description: "The Cypress mono repo"
+        required: false
+        default: '["cypress"]'
+      
+
+  schedule:
+    - cron: '0 17 * * 5'
```
Comment (AtofStryker) [inline .github/workflows/report_weekly_app_kpis.yml]: Is it worth commenting when this runs for those unfamiliar with cron? I thinks its 5 EST on Friday?

**Review by AtofStryker — APPROVED** [review]

File: .github/workflows/report_weekly_app_kpis.yml (hunk 1 above)
Comment (nagash77) [reply .github/workflows/report_weekly_app_kpis.yml]: it's technically 2 EST on fridays.  I originally had it set to '0 14 * * 5' and it was running at 11 AM.  I don't know if the timezone on the workflow runner is different.

Comment (chrisbreiding) [reply .github/workflows/report_weekly_app_kpis.yml]: The workflow is on UTC, so 4/5 hours ahead of EST/EDT respectively.

**Review by chrisbreiding — APPROVED** [review]

File: .github/workflows/report_weekly_app_kpis.yml (hunk 1 above)
Comment (nagash77) [reply .github/workflows/report_weekly_app_kpis.yml]: updated with a comment.  Thanks @chrisbreiding for confirming!

File: .github/workflows/report_weekly_app_kpis.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
+name: 'KPI Report Generation'
```
Comment (mschile) [inline .github/workflows/report_weekly_app_kpis.yml:1]:
```suggestion
name: 'Triage Report'
```

File: .github/workflows/report_weekly_app_kpis.yml
Comment (mschile) [inline .github/workflows/report_weekly_app_kpis.yml:1]: Suggest renaming to `app_triage_weekly_report.yml`

Comment (nagash77) [reply .github/workflows/report_weekly_app_kpis.yml:1]: the end goal is to report on the overall health of the repo and not just firewatch specific.  That just happens to be the first metric I was generating.

**Pull request merged**